p2p/pex/codec.go

```diff
@@ -7,5 +7,5 @@ import (
 var cdc *amino.Codec = amino.NewCodec()
 
 func init() {
-	RegisterPexMessage(cdc)
+	RegisterMessage(cdc)
```

Comment:
```suggestion
	RegisterMessages(cdc)
```

could you rename RegisterMessage to RegisterMessages?